### DISCUSSION TITLE
Adopt juniper-from-schema?

### DISCUSSION THREAD

**Discussion by sebasmagri** [graphql-rust/juniper#1031] [Ideas]
According to [this comment](https://github.com/davidpdrsn/juniper-from-schema/issues/141) @davidpdrsn is up to transfer ownership of the juniper-from-schema project due to lack of time to maintain it. There's some people interested on maintaining it, but as it seems like many people is using it, would the graphql-rust organization be up to adopt it?

I'm looking into some specific use cases that would seem to be easier to implement using it in a new project, but wouldn't like to introduce an unmaintained dependency, mainly considering the evolution pace of juniper itself.

**Comment by tyranron** [comment]
@sebasmagri @davidpdrsn yes, we would like to adopt it, and maybe even merge into Juniper directly in some future.
